.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
+builddir/
```

Comment:
You do not need to do this, meson itself will create a `build/.gitignore` or `builddir/.gitignore` which ignores `*` since it rightly knows whenever it creates a configured build directory under *any* name, and why not just make this work out of the box? :)